### ISSUE TITLE
Can't add my own model readMATmod

### ISSUE THREAD

**Issue by JessicaSingh1997** [euba/BacArena#166]
Error in readMATmod("testLBba.mat") : object 'mod.gpr' not found

I'm trying to upload my own models, and grRules is very well present in my .mat file but I can't figure out why this error is showing up :/

**Comment by JessicaSingh1997** [comment]
I also have the xml files available, if there is an easier way to upload those instead.

**Comment by jotech** [comment]
hi @JessicaSingh1997 can you provide the file `testLBba.mat` then I can try to have a look what's going wrong?

**Comment by JessicaSingh1997** [comment]
Hi Jo, 
Thanks for your response. I tried to use the function with all these .mat files and got the same error. These are metabolic models of S.thermophilis and L.bulgaricus
[Mat_files.zip](https://github.com/euba/BacArena/files/6298474/Mat_files.zip)

**Comment by JessicaSingh1997** [comment]
Hi @jotech can I try adding SMBL models to BacArena?

**Comment by jotech** [comment]
Of course, sbml is even the default way to import models into BacArena :)
```R
library(BacArena)
library(sybilSBML)
mod <- readSBMLmod("model.xml")
bac <- Bac(mod)
arena <- Arena(n=10,m=10)
arena <- addOrg(arena, bac, 5)
arena <- addDefaultMed(arena, bac)
sim <- simEnv(arena, time=3,sec_obj='mtf')
plotGrowthCurve(sim)
plotSubCurve(sim)[1]
```

**Comment by JessicaSingh1997** [comment]
HI @jotech again, I tried getting the packages for making it work this way but it was a dead end. I went though the .mat you used in the example and my .mat files and there are no differences, except maybe a few fields but those are not important to the readMATmod function. 

I'm still getting  object 'mod.gpr' not found as an error even through grRules is a cell in the .mat file. Can you help me out please, its urgent :(
